Sources/SearchableCollectionViewController/SearchableCollectionController.swift

```diff
@@ -1,51 +1,100 @@
 import Foundation
 import UIKit
 
-public class SearchableCollectionController: UIViewController {
-    
-    lazy var collectionView: UICollectionView = {
-        let layout = SearchableCollectionViewLayout()
-        
-        let view = UICollectionView(frame: CGRect.zero, collectionViewLayout: layout)
-        view.translatesAutoresizingMaskIntoConstraints = false
-        view.delegate = self
-        view.backgroundColor = UIColor.white
-        view.contentInset = UIEdgeInsets(top: 0.0, left: 0.0, bottom: SearchBarView.height, right: 0.0)
-        view.showsHorizontalScrollIndicator = false
-        view.showsVerticalScrollIndicator = false
-        
+let SearchBarTag = 123
+
+open class SearchableCollectionController: SweetCollectionController {
+
+    open var searchBar: UISearchBar {
+        return self.searchController.searchBar
+    }
+
+    open lazy var searchController: UISearchController = {
+        let controller = UISearchController(searchResultsController: nil)
+        controller.hidesNavigationBarDuringPresentation = true
+        controller.dimsBackgroundDuringPresentation = false
+
+        controller.searchBar.autoresizingMask = [.flexibleWidth]
+
+        return controller
+    }()
+
+    lazy var searchBarContainerView: UIView = {
```

Comment:
was even considering making this open, but I think we can make it private for now. 🆗